api/src/services/listing.service.ts

```diff
@@ -163,7 +163,15 @@ export class ListingService implements OnModuleInit {
   onModuleInit() {
     startCronJob(
       this.prisma,
-      CRON_JOB_NAME,
+      LISTING_CRON_JOB_NAME,
+      process.env.LISTING_PROCESSING_CRON_STRING,
+      this.process.bind(this),
+      this.logger,
+      this.schedulerRegistry,
+    );
+    startCronJob(
+      this.prisma,
+      LOTTERY_CRON_JOB_NAME,
       process.env.LISTING_PROCESSING_CRON_STRING,
```

Comment:
Should this be the lottery string?

Reply:
I'm not sure what you are asking here. There are two calls to startCronJob, one for listing process and on for lottery expiration. Each calls using the respective name

Reply:
For this specific line it's using `LISTING_PROCESSING_CRON_STRING` when maybe it should be using the lottery one

Reply:
I was confused by the github comment layout lol corrected